BREAKING_CHANGES.md

```diff
@@ -1,6 +1,17 @@
 ## Unreleased
 
 *   When using a commit message that matches **both** `*-version-bump-message` and `no-bump-message`, there is no increment for that commit. In other words, `no-bump-message` now takes precedence over `*-version-bump-message`.
+*   The fallback version strategy now returns `0.0.0` and is flagged with `ShouldIncrement` equal to `true`. This yields the version `0.1.0` on the `develop` branch (`IncrementStrategy.Minor` by default) and `0.0.1` on the `main` branch (`IncremetnStrategy.Patch` by default).
+*   The current branch (child) inherits its configuration from the source (parent) branch if the `increment` strategy is set to `Inherit`. This makes branch configuration recursive, simpler, more intuitive, more flexible, and more robust.
+*   Instead of having a single effective configuration, we now have one effective configuration per branch where the increment strategy is not set to `inherit`.
+*   The new implementation of the branch configuration inheritance affects per default only the pull-requests, hotfix and feature branches. In this case the next version will be generated like the child branch is not existing and the commits have been made on the source branch.
+    *   The following example illustrates this behavior. On the feature branch the semantic version 1.1.0-just-a-test.1+2 will be generated instead of version 1.0.0-just-a-test.1+3:
```

Comment:
```suggestion
    *   The following example illustrates this behavior. On the feature branch the semantic version `1.1.0-just-a-test.1+2` will now be generated instead of version `1.0.0-just-a-test.1+3` previously:
```